server/apps/publish/content/correct.py

```diff
@@ -26,6 +26,7 @@ def on_update(self, updates, original):
         ArchiveCropService().validate_multiple_crops(updates, original)
         super().on_update(updates, original)
         updates[ITEM_OPERATION] = ITEM_CORRECT
+        updates['versioncreated'] = utcnow()
```

Comment:
I think this should not happen when you publish, but only when you kill and correct